code/datums/traits/good.dm

```diff
@@ -108,3 +108,16 @@
 	mob_trait = TRAIT_VORACIOUS
 	gain_text = "<span class='notice'>You feel HONGRY.</span>"
 	lose_text = "<span class='danger'>You no longer feel HONGRY.</span>"
+/datum/trait/hlumi_eyes
+	name = "High Luminosity Eyes"
+	desc = "You decided to upgrade your weak fleshy eyes with these cool cybernetic ones, your vision is augumented!"
+	value = 1
+
+/datum/trait/hlumi_eyes/on_spawn()
+	var/mob/living/carbon/human/H = trait_holder
+	var/obj/item/organ/eyes/robotic/glow = new
+	glow.Insert(H, drop_if_replaced = FALSE)
```

Comment:
Not sure how organ code works, but make sure this deletes the other eyes.

Reply:
I believe eyes default to `drop_if_replaced` so they should be ok.

Only testing will tell though wink wink

Reply:
Hehe, wink wink. Cuz they're eyes.

Reply:
*rolls eyes*

Reply:
/glow here is a variable name not part of the path, add /G and replace `glow.` on the next line with `G.`